frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkerThread.java

```diff
@@ -147,6 +149,8 @@ public void run() {
         } catch (Throwable t) {
             logger.warn("Backend worker thread exception.", t);
         } finally {
+            logger.info("Shutdown latch called");
```

Comment:
We don't need latch if we retry at the end of this block

Reply:
Makes sense. Changed it.